scripts/obj_trimmer.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) 2022 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+import argparse
+import os
+from typing import Optional, List
+
+"""
+    Used to reduce the size of obj files used for printer platform models.
+    
+    Trims trailing 0 from coordinates
+    Removes duplicate vertex texture coordinates
+    Removes any rows that are not a face, vertex or vertex texture
+"""
+
+def process_obj(input_file, output_file):
+    with open(input_file, "r") as in_obj, open("temp", "w") as temp:
+        trim_lines(in_obj, temp)
+
+    with open("temp", "r") as temp, open(output_file, "w") as out_obj:
+        merge_duplicate_vt(temp, out_obj)
+
+    os.remove("temp")
+
+
+def trim_lines(in_obj, out_obj):
```

Comment:
Missing typing.
```suggestion
def trim_lines(in_obj: TextIO, out_obj: TextIO) -> None:
```
(Be sure to add those types to the import as well (from the `typing` library).